app/views/home/_fandoms.html.erb

```diff
@@ -0,0 +1,10 @@
+<ul class="menu">
+  <li><%= link_to t(".all_fandoms"), media_path %></li>
+  <% cache "homepage-fandoms-version1", skip_digest: true do %>
+    <% Media.for_menu.each do |medium| %>
+      <% unless medium.id.nil? %>
+        <li id="medium_<%= medium.id %>"><%= link_to t(".#{medium.translation_key}"), medium_fandoms_path(medium) %></li>
```

Comment:
Can you take the `id` attribute off? I'm not really sure what we use it for, if anything, in the dropdown menus, but having the same `id` in both places makes for invalid HTML because IDs have to be unique. I just made a mess all over the place with my genius reuse of that partial. 🙃